examples/colors_rgb.rs

```diff
@@ -1,72 +1,130 @@
 /// This example shows the full range of RGB colors that can be displayed in the terminal.
 ///
 /// Requires a terminal that supports 24-bit color (true color) and unicode.
+///
 use std::{
-    error::Error,
-    io::{stdout, Stdout},
-    rc::Rc,
-    time::Duration,
+    io::{stdout, Write},
+    time::{Duration, Instant},
 };
 
+use anyhow::Result;
 use crossterm::{
     event::{self, Event, KeyCode, KeyEventKind},
     terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
     ExecutableCommand,
 };
-use palette::{
-    convert::{FromColorUnclamped, IntoColorUnclamped},
-    Okhsv, Srgb,
-};
+use palette::{convert::FromColorUnclamped, Okhsv, Srgb};
 use ratatui::{prelude::*, widgets::*};
 
-type Result<T> = std::result::Result<T, Box<dyn Error>>;
-
 fn main() -> Result<()> {
-    install_panic_hook();
-    App::new()?.run()
+    // stdout runs ~ 2x faster than stderr on a 2023 Macbook Pro M2 Max (~22-26fps vs ~11-12fps)
```

Comment:
Oh that's interesting, `stdout` is also faster on my machine 
- from ~1.60 to ~2 fps in fullscreen
- from ~15 to ~90 fps in 80x24!

If this is consistent, we might want to keep that in mind. We currently use stderr in the template. And we should mention that in the [FAQ](https://ratatui.rs/faq/#should-i-use-stdout-or-stderr)

Reply:
Yep - definitely agree. I've been slowly changing things to stdout in various example code etc.

Reply:
https://github.com/ratatui-org/website/issues/274